developer-docs/upgrading_kubernetes.md

```diff
@@ -233,6 +233,30 @@ Every RKE2 release, from RC to primary, triggers the release process of the down
   - [Repository](https://github.com/rancher/rke2-upgrade)
   - [Drone publish job](https://drone-publish.rancher.io/rancher/rke2-upgrade)
 
+## Prepare for an 'R2'
+At this point it is likely that the release is over. However, in the event of a CVE, critical bug affecting user experiences or something similiar, we may be required to cut what the release team refers to as an 'R2' or 'Release 2'. This process is identical to the release process, save only that the tags are cut with ```+rke2r2```. To prep an R2 one need only to perform the following:
+
+Create a new release tag at the [image-build-kubernetes](https://github.com/rancher/image-build-kubernetes) repo.
+
+* Click "Releases"
+* Click "Draft a new release"
+* Enter the new release version (the RKE2 Kubernetes version), appended with `-buildYYYYMMdd`, into the "Tag version" box.  **NOTE** The build system is in UTC. The command `TZ=utc date '+-build%Y%m%d'` can be used to get the correct format.
+* When converting the RKE2 version to the Kubernetes version, use dash instead of plus, and do not include any alpha/beta/rc components. For example, if preparing for RKE2 `v1.23.5+rke2r2` before 5 PM Pacific on Friday, August 27th 2021 you would tag `v1.23.5-rke2r2-build20210829`
+* Click the "Publish release" button. 
+
+This will take a few minutes for [CI](https://drone-pr.rancher.io/rancher/image-build-kubernetes) to run but upon completion, a new image will be available in [Dockerhub](https://hub.docker.com/r/rancher/hardened-kubernetes).
+
+Then:
+
+* Dockerfile: `FROM rancher/hardened-kubernetes:v1.23.5-rke2r2-build20220217 AS kubernetes`
+* version.sh:
+  * `KUBERNETES_VERSION=${KUBERNETES_VERSION:-v1.23.5}`
+  * `KUBERNETES_IMAGE_TAG=${KUBERNETES_IMAGE_TAG:-v1.21.13-rke2r2-build20220525}`
+* In v1.21, kube-proxy is still in use an needs to be updated in the Dockerfile. Update the line: `RUN CHART_VERSION="v1.21.10-build2021041301"     CHART_FILE=/charts/rke2-kube-proxy.yaml` to the relevant RKE2 Kubernetes build version.
```

Comment:
I don't think this line is relevant for prepping an R2. Maybe remove it, or remove the whole section and only refer to the Hardened Kubernetes section above.